mne/decoding/tests/test_time_gen.py

```diff
@@ -258,18 +293,21 @@ def test_generalization_across_time():
 
     # Test that gets error if train on one dataset, test on another, and don't
     # specify appropriate cv:
-    gat = GeneralizationAcrossTime()
+    if check_version('sklearn', '0.18'):
```

Comment:
line 56 in test_time_gen.py Travis runs with multiple versions; the error thus breaks one of the version.

Let me know if I should add something.